api/src/main/java/org/openmrs/module/rwandahivflowsheet/impl/pih/ProphylaxisMapping.java

```diff
@@ -111,23 +112,20 @@ public boolean getProphylaxisOther() {
 		public String getProphylaxisTypeOther(){
 			if (drugOrder != null && drugOrder.getConcept() != null 
 					&& !isCotrimoxisole() && !isFluconazole() && !isDapsone())
-				return drugOrder.getConcept().getBestName(Context.getLocale()).getName();
+				return drugOrder.getConcept().getDisplayString();
 			return "";
 		}
 		
 		public Date getStopDate(){
 			if (drugOrder != null){
-				if (drugOrder.getDiscontinuedDate() != null)
-					return drugOrder.getDiscontinuedDate();
-				if (drugOrder.getAutoExpireDate() != null)
-					return drugOrder.getAutoExpireDate();
+				return drugOrder.getEffectiveStopDate();
 			}
 			return null;
 		}
 		
 		public Date getStartDate(){
 			if (drugOrder != null)
-				return drugOrder.getStartDate();
```

Comment:
Do we know that dateActivated is right here, and not effectiveStartDate?  Can we discuss?

Reply:
This seems to be used throughout this PR, and I think I saw that @dkayiwa recommended this based on a thread somewhere.  I'd be interested to learn more.